modules/client-configuration/pages/supported-features-centos.adoc

```diff
@@ -20,51 +20,48 @@ The icons in this table indicate:
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| System redeployment ({ay}) | ``Traditional`` | icon:check[role="green"] | ``Salt``
 | Contact methods | ``Traditional``: OSAD, RHNSD, SSH-push. +
-``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. +
-``Salt``: ZeroMQ, Salt-SSH
-| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"]
-| Action chains | icon:check[role="green"] | icon:check[role="green"]
-| Software crash reporting | icon:times[role="danger"] | ``Traditional``
-| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"]
-| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"]
-| CVE auditing |  icon:check[role="green"] | icon:check[role="green"]
-| SCAP auditing | icon:check[role="green"] | icon:check[role="green"]
-| Package verification | ``Traditional`` | ``Traditional``
-| Package locking | ``Traditional`` | ``Traditional``
-| System locking |  ``Traditional`` | ``Traditional``
-| System snapshot | ``Traditional`` | ``Traditional``
-| Configuration file management |  icon:check[role="green"] | icon:check[role="green"]
-| Snapshots and profiles |  ``Traditional``. +
-``Salt``: Profiles supported, Sync not supported | ``Traditional``. +
-``Salt``: Profiles supported, Sync not supported
-| Power management |  icon:check[role="green"] | icon:check[role="green"]
-| Monitoring |  ``Salt`` | ``Salt``
-| Docker buildhost |  icon:times[role="danger"] | icon:times[role="danger"]
-| Build Docker image with OS | icon:times[role="danger"] | icon:times[role="danger"]
-| Kiwi buildhost |  icon:times[role="danger"] | icon:times[role="danger"]
-| Build Kiwi image with OS |  icon:times[role="danger"] | icon:times[role="danger"]
+``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. ``Salt``: ZeroMQ, Salt-SSH | ``Salt``: ZeroMQ, Salt-SSH
+| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Action chains | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Software crash reporting | icon:times[role="danger"] | ``Traditional`` | icon:times[role="danger"]
+| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| CVE auditing |  icon:check[role="green"] | icon:check[role="green"] | ``Salt``
```

Comment:
For all CentOS versions: this requires errata data with CVE IDs. (see also "Apply patches")